src/index.html

```diff
@@ -6,6 +6,12 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Toto asi nepatrí do tohoto PR či?

Reply:
Nie nepatri, to je tag manazer. Ale mozeme to tam mat, potom iba rozsitim o dalsie veci